packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -327,6 +327,189 @@ export interface CommonAutoScalingGroupProps {
   readonly terminationPolicies?: TerminationPolicy[];
 }
 
+/**
+ * MixedInstancesPolicy allows you to configure a group that diversifies across On-Demand Instances
+ * and Spot Instances of multiple instance types. For more information, see Auto Scaling groups with
+ * multiple instance types and purchase options in the Amazon EC2 Auto Scaling User Guide:
+ *
+ * https://docs.aws.amazon.com/autoscaling/ec2/userguide/asg-purchase-options.html
+ */
+export interface MixedInstancesPolicy {
```

Comment:
I am wondering if it is better to have a separate file for mixed instance policy because this file is getting bigger.